src/components/CourseChecklist/index.jsx

```diff
@@ -124,30 +126,141 @@ class CourseChecklist extends React.Component {
   getListItems = () => (
     this.state.checks.map((check) => {
       const isCompleted = this.isCheckCompleted(check.id);
+      const shouldShowCommentSection = this.shouldShowCommentSection(check.id);
 
       return (
         <div
           className={classNames(
-            'bg-white border my-1 py-4',
+            'bg-white border my-2',
+            { 'pt-4': shouldShowCommentSection, 'py-4': !shouldShowCommentSection },
             this.getChecklistItemColorClassName(isCompleted),
             styles['checklist-item'])
           }
           id={`checklist-item-${check.id}`}
           key={check.id}
         >
-          <div className="row no-gutters">
-            {this.getCompletionIcon(check.id)}
+          <div className="align-items-center no-gutters row">
+            <div className="col-1">
+              {this.getCompletionIcon(check.id)}
+            </div>
             <div className="col">
               {this.getShortDescription(check.id)}
               {this.getLongDescription(check.id)}
             </div>
             {this.shouldShowUpdateLink(check.id) ? this.getUpdateLink(check.id) : null}
           </div>
+          {shouldShowCommentSection ? this.getCommentSection(check.id) : null}
         </div>
       );
     })
   );
 
+  getCommentSectionIcon = () => (
+    <div className="text-center">
+      <Icon
+        className={[classNames(
+          FontAwesomeStyles.fa,
+          FontAwesomeStyles['fa-lg'],
+          FontAwesomeStyles['fa-comment'],
+          styles['comment-icon'],
+        )]
+        }
+      />
+    </div>
+  );
+
+  getGradingPolicyCommentSection = () => (
+    this.getComment(
+      <WrappedMessage
+        message={messages.gradingPolicyComment}
+        values={{
+          percent: <FormattedNumber
+            maximumFractionDigits={2}
+            minimumFractionDigits={2}
+            value={(this.props.data.grades.sum_of_weights * 100).toFixed(2)}
+          />,
+        }}
+      />,
+    )
+  );
+
+  getAssignmentDeadlineCommentSection = () => {
+    const gradedAssignmentsOutsideDateRange = [].concat(
+      this.props.data.assignments.graded.assignments_with_dates_before_start,
+      this.props.data.assignments.graded.assignments_with_dates_after_end,
+    );
+
+    const message = (
+      <React.Fragment>
+        <WrappedMessage message={messages.assignmentDeadlinesComment} />
+        <ul className={styles['assignment-list']}>
```

Comment:
To make sure that I understand - in the DOM this is a unordered list, but the CSS is rendering it as a comma separated list - right?